test/membrane_http_adaptive_stream/integration_test/sink_bin_integration_test.exs

```diff
@@ -0,0 +1,192 @@
+defmodule Membrane.HTTPAdaptiveStream.SinkBinIntegrationTest do
+  use ExUnit.Case, async: true
+
+  import Membrane.Testing.Assertions
+  alias Membrane.{Testing}
+  alias Membrane.Hackney.Source
+
+  @create_references_mode false
+
+  @single_video_track_source [
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/ffmpeg-testsrc.h264",
+       hackney_opts: [follow_redirect: true]
+     }, :H264, "single_video_track"}
+  ]
+  @single_video_track_ref_path "./test/membrane_http_adaptive_stream/integration_test/reference_playlists/single_video_track/"
+  @single_video_track_test_path "./test/membrane_http_adaptive_stream/integration_test/test_playlists/single_video_track/"
+
+  @audio_video_tracks_sources [
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/test-audio.aac",
+       hackney_opts: [follow_redirect: true]
+     }, :AAC, "audio_track"},
+    {%Source{
+       location:
+         "https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/ffmpeg-testsrc.h264",
+       hackney_opts: [follow_redirect: true]
+     }, :H264, "video_track"}
+  ]
+  @audio_video_tracks_ref_path "./test/membrane_http_adaptive_stream/integration_test/reference_playlists/audio_video_tracks/"
+  @audio_video_tracks_test_path "./test/membrane_http_adaptive_stream/integration_test/test_playlists/audio_video_tracks/"
+
+  @audio_multiple_video_tracks_sources [
+    {%Source{
```

Comment:
This could be easily simplified having 3 elements tuples and doing the map operation on them, this takes too much space

Reply:
Done.